.github/workflows/release.yml

```diff
@@ -99,3 +110,26 @@ jobs:
           repo_token: ${{ secrets.GITHUB_TOKEN }}
           tag: ${{ github.ref }}
           file: claudie_checksums.txt
+
+      - name: Set up git author
```

Comment:
I'm not fully sure, but it might be nicer if we made a new file, something ike `release-docs.yml`, to not mix it all into one file

Reply:
In my opinion, this is a really good idea. Can I implement it?

Reply:
yes please